Dockerfiles/test_suite-al2023

```diff
@@ -0,0 +1,25 @@
+# This Dockerfile is a minimal example for a RHEL-based SSG test suite target container.
+FROM amazonlinux:2023
+
+ENV AUTH_KEYS=/root/.ssh/authorized_keys
+
+ARG CLIENT_PUBLIC_KEY
+ARG ADDITIONAL_PACKAGES
+
+# Install Python so Ansible remediations can work
+# Don't clean all, as the test scenario may require package install.
+RUN true \
+        && dnf install -y openssh-clients openssh-server openscap-scanner \
```

Comment:
The problem was that the Automatus script needs the tar command to be present in the container.

This change fixed it for me:

```suggestion
        && dnf install -y openssh-clients openssh-server openscap-scanner tar \
```